packages/devtools_app/lib/src/service/service_manager.dart

```diff
@@ -93,6 +93,7 @@ class ServiceConnectionManager {
   late final ServiceExtensionManager _serviceExtensionManager;
 
   ConnectedApp? connectedApp;
+  AppState appState = AppState();
```

Comment:
Seems odd to have an appState object before there is actually a connected app. I would keep this consistent with other data in the class that is null unless there is actually an app.